synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -192,10 +201,29 @@ export class SynopsysBridge {
         formattedCommand,
         tempDir
       );
+    } else if (SCAN_TYPE == "srm") {
+      [formattedCommand, errors] = await this.prepareSrmCommand(
+        formattedCommand,
+        tempDir
+      );
     }
     return [formattedCommand, errors];
   }
 
+  private async prepareSrmCommand(
+    formattedCommand: string,
+    tempDir: string
+  ): Promise<[string, string[]]> {
+    const srmErrors: string[] = validateSrmInputs();
+    const commandFormatter = new SynopsysToolsParameter(tempDir);
```

Comment:
maybe we can move` const commandFormatter = new SynopsysToolsParameter(tempDir);` into if block after L219